x-pack/plugins/session_view/public/components/SessionView/index.tsx

```diff
@@ -47,6 +41,13 @@ interface ProcessEventResults {
   };
 }
 
+const BUTTON_OPEN_DETAIL_PANEL = i18n.translate(
```

Comment:
Did we want to standardize on this for our team? I think so far I've been using FormattedMessage component. I guess that might not work so well if you need just the string value though.

Reply:
the [guideline](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md#in-reactjs) suggests it's recommended to use `FormattedMessage`, and only use `i18n.translate` when we have to get a string. I will update this to use `FormattedMessage`.